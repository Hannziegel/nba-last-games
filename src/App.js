import { BrowserRouter, Routes, Route } from 'react-router-dom';
import Navbar from './components/Navbar';
import Teams from './components/Teams';
import TeamLastGames from './components/TeamLastGames';
import './styles/App.css';

function App() {
  return (
    <BrowserRouter>
      <Navbar />
      <Routes>
        <Route path="/" element={<Teams />} />
        <Route path="/teams" element={<Teams />} />
        <Route path="/team-last-games" element={<TeamLastGames />} />
      </Routes>
    </BrowserRouter>
  );
}

export default App;
