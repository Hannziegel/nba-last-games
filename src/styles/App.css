.App {
  text-align: center;
}

.test {
  /* height: 40vmin; */
  height: 5vw;
  width: 5vw;
  position: fixed;
  top: 40%;
  left: 50%;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .test {
    animation: rotate360 infinite 5s linear;
  }
}

@keyframes rotate360 {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
}
