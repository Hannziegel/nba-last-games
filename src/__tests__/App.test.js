import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';

import store from '../redux/configureStore';
import App from '../App';

describe('App testing', () => {
  test('snapshot for App', () => {
    const tree = render(
      <React.StrictMode>
        <Provider store={store}>
          <App />
        </Provider>
      </React.StrictMode>,
    );
    expect(tree).toMatchSnapshot();
  });
});
