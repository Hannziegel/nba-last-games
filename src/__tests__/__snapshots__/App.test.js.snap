// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App testing snapshot for App 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="nav container"
      >
        <div
          class="nav-configurations"
        >
          <img
            alt="League"
            class="league-img"
            src="nba-logo-2.png"
          />
        </div>
        <a
          class="arrow-back"
          href="/teams"
        >
          <svg
            class="icon-arrow-back"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M48 256c0 114.87 93.13 208 208 208s208-93.13 208-208S370.87 48 256 48 48 141.13 48 256zm212.65-91.36a16 16 0 01.09 22.63L208.42 240H342a16 16 0 010 32H208.42l52.32 52.73A16 16 0 11238 347.27l-79.39-80a16 16 0 010-22.54l79.39-80a16 16 0 0122.65-.09z"
            />
          </svg>
        </a>
      </nav>
      <div
        class="page-container container"
      >
        <div
          class="league-container"
        >
          <div
            class="league-info-container"
          >
            <h2
              class="league-name"
            >
              Loading
            </h2>
            <p
              class="league-number-teams"
            >
              1
               Teams
            </p>
          </div>
        </div>
        <input
          class="searchBar"
          placeholder="Search..."
          type="search"
          value=""
        />
        <div
          class="teams-container container"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <nav
      class="nav container"
    >
      <div
        class="nav-configurations"
      >
        <img
          alt="League"
          class="league-img"
          src="nba-logo-2.png"
        />
      </div>
      <a
        class="arrow-back"
        href="/teams"
      >
        <svg
          class="icon-arrow-back"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 512 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M48 256c0 114.87 93.13 208 208 208s208-93.13 208-208S370.87 48 256 48 48 141.13 48 256zm212.65-91.36a16 16 0 01.09 22.63L208.42 240H342a16 16 0 010 32H208.42l52.32 52.73A16 16 0 11238 347.27l-79.39-80a16 16 0 010-22.54l79.39-80a16 16 0 0122.65-.09z"
          />
        </svg>
      </a>
    </nav>
    <div
      class="page-container container"
    >
      <div
        class="league-container"
      >
        <div
          class="league-info-container"
        >
          <h2
            class="league-name"
          >
            Loading
          </h2>
          <p
            class="league-number-teams"
          >
            1
             Teams
          </p>
        </div>
      </div>
      <input
        class="searchBar"
        placeholder="Search..."
        type="search"
        value=""
      />
      <div
        class="teams-container container"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
